batch2/batch/driver/instance.py

```diff
@@ -104,6 +106,36 @@ def adjust_free_cores_in_memory(self, delta_mcpu):
         self._free_cores_mcpu += delta_mcpu
         self.instance_pool.adjust_for_add_instance(self)
 
+    @property
+    def failed_request_count(self):
+        return self._failed_request_count
+
+    async def mark_healthy(self):
+        now = time.time()
+        await self.db.execute_update(
+            '''
+UPDATE instances
+SET last_updated = %s,
+  failed_request_count = 0
+WHERE id = %s;
+''',
+            (now, self.id))
+
+        self.instance_pool.adjust_for_remove_instance(self)
+        self._failed_request_count = 0
+        self._last_updated = now
+        self.instance_pool.adjust_for_add_instance(self)
+
+    async def incr_failed_request_count(self):
+        now = time.time()
```

Comment:
This function name is confusing me. Copy and paste error potentially? The function is setting the failed request count to 0, not incrementing it.

Reply:
Oops, yes, this was totally wrong.  Fixed, thanks!